test/invoice_item_test.rb

```diff
@@ -7,7 +7,7 @@ class InvoiceItemTest < MiniTest::Test
 
   def setup
     sales_engine = SalesEngine.new
-    @parent = InvoiceItemRepository.new('test/support/sample_invoice_items.csv', sales_engine)
+    @parent = InvoiceItemRepository.new('test/support/invoice_items.csv', sales_engine)
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.